packages/fxa-auth-server/test/local/routes/subscriptions/paypal.js

```diff
@@ -181,6 +181,8 @@ describe('subscriptions payPalRoutes', () => {
       payPalHelper.agreementDetails = sinon.fake.resolves({
         countryCode: 'CA',
       });
+      stripeHelper.customerTaxId = sinon.fake.returns(false);
```

Comment:
nit: This works for the purpose of the tests since `customerTaxId` is only used in `if` conditions so far, but it doesn't really match the type of the actual function.

Reply:
Updated.